test/integration/test_groundlight.py

```diff
@@ -252,6 +252,7 @@ def test_submit_image_query_with_human_review_param(gl: Groundlight, detector: D
         assert is_valid_display_result(_image_query.result)
 
 
+@pytest.mark.skip_for_edge_endpoint(reason="The edge-endpoint does not support passing metadata.")
```

Comment:
As we were mentioned yesterday I think, we can also easily support this once we have persistent storage for image queries on the edge.

Reply:
Yeah but for now we want to unblock other PRs on the edge. And in general we may not be fully feature complete on the edge relative to the cloud.